solidity/contracts/converter/types/standard-pool/StandardPoolConverter.sol

```diff
@@ -8,6 +8,7 @@ import "../../ConverterVersion.sol";
 import "../../interfaces/IConverter.sol";
 import "../../interfaces/IConverterAnchor.sol";
 import "../../interfaces/IConverterUpgrader.sol";
+import "../../../INetworkSettings.sol";
```

Comment:
General reminder: don't forget to format the code before the final submission

Reply:
Any idea why scoping a block of code with `/* eslint-disable max-len */` and `/* eslint-enable max-len */` doesn't do what it's supposed to do?

Reply:
It might require `//` here, but in any case - try to disable the rule for a single line first via `// eslint-disable-next-line max-len`, so that it won't "infect" unrelated lines